plugins/techdoc-expandable-toc/src/addons/ExpandableToc.tsx

```diff
@@ -68,13 +62,17 @@ export const ExpandableTocAddon = () => {
     return (
         <>
             { !isEmpty ? (
-                <StyledButton
-                    size="small"
+                <span
+                    className={classes.span}
                     onClick={handleState}
+                    onKeyDown={handleState}
                     aria-label={expanded?.expandToc ? 'collapse-toc' : 'expand-toc'}
                 >
-                    {expanded?.expandToc ? <ExpandedIcon /> : <CollapsedIcon />}
-                </StyledButton>
+                    <ExpandedIcon 
+                      style={{ transform: expanded?.expandToc ? 'rotate(90deg)' : 'rotate(0)' }} 
+                      cursor='pointer'
+                    />
+                </span>
             ) : null}
         </>
```

Comment:
I honestly don't know enough about css to comment on these changes. From what we discussed the other day, this approach sounds reasonable. It sounds like you tested it and it worked as expected, so I'm fine with that. 

Maybe Jen or Shea could comment more on the css part.